web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -123,6 +132,67 @@ be edited, its `Revision: X` label will be increased by one, and a note will be
 to the `PDEP-N history` section. This will let readers understand that the PDEP has
 changed and avoid confusion.
 
+## <a id="List-of-issues"></a> List of issues that could have been PDEPs for context
+### Clear examples for potential PDEPs:
+- Adding a new parameter to many existing methods, or deprecating one in many places. For example:
+  - The `numeric_only` deprecation affected many methods and could have been a PDEP.
+- Adding a new data type has impact on a variety of places that need to handle the data type.
+  Such wide-ranging impact would require a PDEP. For example:
+  - `Categorical` ([GH-7217][7217], [GH-8074][8074]), `StringDtype` ([GH-8640][8640]), `ArrowDtype`
+- A significant (breaking) change in existing behavior. For example:
+  - Copy/view changes ([GH-36195][36195])
+- Support of new python features with a wide impact on the project. For example:
+  - Supporting typing within pandas vs. creation of `pandas-stubs` ([GH-43197][43197],
+    [GH-45253][45253])
+- New required dependency
+
+### Borderline examples:
+- Small changes to core functionality, such as `DataFrame` and `Series`, should always be considered
```

Comment:
This first bullet point should maybe be a normal paragraph, and only the remaining bullet points as a list? (this bullet point also ends with a `:`, so I would expect the other points to be a sublist of this one, otherwise)